pricegraph/src/lib.rs

```diff
@@ -84,6 +84,37 @@ impl TransitiveOrder {
     }
 }
 
+/// A struct representing a market.
+///
+/// This is used for computing transitive orderbooks.
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
+pub struct Market {
+    /// The base token being purchased in a market.
```

Comment:
Technically the base token can also be sold in a market, against the bids.

Reply:
I really wasn't sure how to describe base and quote tokens... I'll look up a Wikipedia definition or something.

Reply:
I'm going to use language from: https://en.wikipedia.org/wiki/Currency_pair